pkg/cmd/attestation/verification/extensions.go

```diff
@@ -0,0 +1,28 @@
+package verification
+
+import (
+	"fmt"
+	"strings"
+)
+
+func VerifyCertExtensions(results []*AttestationProcessingResult, owner string, repo string) error {
+	for _, attestation := range results {
+		// TODO: handle proxima prefix
+		expectedSourceRepositoryOwnerURI := fmt.Sprintf("https://github.com/%s", owner)
+		sourceRepositoryOwnerURI := attestation.VerificationResult.Signature.Certificate.Extensions.SourceRepositoryOwnerURI
+		if sourceRepositoryOwnerURI != "" && !strings.EqualFold(expectedSourceRepositoryOwnerURI, sourceRepositoryOwnerURI) {
+			return fmt.Errorf("expected SourceRepositoryOwnerURI to be %s, got %s", expectedSourceRepositoryOwnerURI, sourceRepositoryOwnerURI)
+		}
+
+		// if repo is set, check the SourceRepositoryURI field
+		if repo != "" {
+			// TODO: handle proxima prefix
+			expectedSourceRepositoryURI := fmt.Sprintf("https://github.com/%s", repo)
+			sourceRepositoryURI := attestation.VerificationResult.Signature.Certificate.Extensions.SourceRepositoryURI
+			if sourceRepositoryURI != "" && !strings.EqualFold(expectedSourceRepositoryURI, sourceRepositoryURI) {
```

Comment:
Same question as above, for `sourceRepositoryURI`.

Reply:
make sense. remove the logic of checking sourceRepositoryURI is empty.